File: PlataformaVirtualHumanos/src/app/_services/MisCancionesServices/CancionModel.ts
export class CancionModel
{
    id:any;
    titulo:any;
    genero:any;
    artista:any;
    fecha:any;
    descripcion:any;
    imagenPortada:any;
    constructor(){
    }
    set_id(id:any){
        this.id=id;
    }
    get_id(){
        return this.id
    }
    set_titulo(titulo:any){
        this.titulo=titulo;
    }
    get_titulo(){
        return this.titulo
    }
    set_genero(genero:any){
        this.genero=genero;
    }
    get_genero(){
        return this.genero
    }
    set_artista(artista:any){
        this.artista=artista;
    }
    get_artista(){
        return this.artista
    }
    set_fecha(fecha:any){
        this.fecha=fecha;
    }
    get_fecha(){
        return this.fecha
    }
    set_descripcion(descripcion:any){
        this.descripcion=descripcion;
    }
    get_descripcion(){
        return this.descripcion
    }
    set_imagenPortada(imagenPortada:any){
        this.imagenPortada=imagenPortada;
    }
    get_imagenPortada(){
        return this.imagenPortada
    }
}

File: Proyecto_APK_Movil/app/src/main/java/com/example/proyecto_apk/MisCanciones/CancionModel.java
package com.example.proyecto_apk.MisCanciones;

import android.content.Context;

import com.example.proyecto_apk.ConeccionConElServidor.ConeccionServidor;
import com.example.proyecto_apk.PacienteLogueado;
import com.example.proyecto_apk.data.model.Paciente;

import org.json.JSONException;
import org.json.JSONObject;

import java.util.Calendar;

public class CancionModel {
  String id;
  String titulo;
  String genero;
  String artista;
  Calendar fecha;
  String descripcion;
  String imagenPortada;

  public CancionModel() {
  }

  public String getId() {
    return id;
  }

  public void setId(String id) {
    this.id = id;
  }

  public String getTitulo() {
    return titulo;
  }

  public void setTitulo(String titulo) {
    this.titulo = titulo;
  }

  public String getGenero() {
    return genero;
  }

  public void setGenero(String genero) {
    this.genero = genero;
  }

  public String getArtista() {
    return artista;
  }

  public void setArtista(String artista) {
    this.artista = artista;
  }

  public Calendar getFecha() {
    return fecha;
  }

  public void setFecha(Calendar fecha) {
    this.fecha = fecha;
  }

  public String getDescripcion() {
    return descripcion;
  }

  public void setDescripcion(String descripcion) {
    this.descripcion = descripcion;
  }

  public String getImagenPortada() {
    return imagenPortada;
  }

  public void setImagenPortada(String imagenPortada) {
    this.imagenPortada = imagenPortada;
  }


}
